extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -387,9 +374,25 @@ export class Bind {
       }
       const element = dev().assertElement(node);
       const tagName = element.tagName;
-      if (DYNAMIC_TAGS[tagName]) {
-        this.observeElementForMutations_(element);
+
+      let dynamicElements = [];
+      if (typeof element.getDynamicElements === 'function') {
+        dynamicElements = element.getDynamicElements();
+      } else if (element.tagName === 'FORM') {
+        // FORM is not an amp element, so it doesn't have the getter.
```

Comment:
Done.

When I tried to add the type annotation 

`@return {?../extensions/amp-form/0.1/amp-form.AmpForm}`

on the functions in `src/form.js` I got the following error:

`WARNING - Failed to load module "../extensions/amp-form/0.1/amp-form.js"`

Any idea what's happening here?